src/www/ui/page/AdminGroupEdit.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/*
+ SPDX-FileCopyrightText: © 2025 Siemens AG
+
+ SPDX-License-Identifier: GPL-2.0-only
+*/
+
+namespace Fossology\UI\Page;
+
+use Fossology\Lib\Auth\Auth;
+use Fossology\Lib\Dao\UserDao;
+use Fossology\Lib\Plugin\DefaultPlugin;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * \class AdminGroupEdit extends DefaultPlugin
+ * \brief edit group name
+ */
+class AdminGroupEdit extends DefaultPlugin
+{
+
+  const NAME = 'group_edit';
+
+  function __construct()
+  {
+    parent::__construct(self::NAME, array(
+        self::TITLE => _("Edit Group"),
+        self::MENU_LIST => "Admin::Groups::Edit Group",
+        self::PERMISSION => Auth::PERM_ADMIN,
+        self::REQUIRES_LOGIN => TRUE
+    ));
+  }
+
+  /**
+   * @param Request $request
+   * @return Response
+   */
+  protected function handle(Request $request)
+  {
+    $userId = Auth::getUserId();
+    $vars = array();
+
+    /** @var UserDao $userDao */
+    $userDao = $this->getObject('dao.user');
+    $groupMap = $userDao->getDeletableAdminGroupMap($userId,
+      $_SESSION[Auth::USER_LEVEL]);
+
+    $groupId = $request->get('grouppk');
+    $newGroupName = $request->get('newgroupname');
+    if (! empty($groupId) && ! empty($newGroupName)) {
+      try {
+        $userDao->editGroup($groupId, $newGroupName);
+        $vars['message'] = _("Group") . ' ' . $groupMap[$groupId] . ' ' . _("edited to ".$newGroupName ) . '.';
+        $groupMap[$groupId] = $newGroupName;
+      } catch (\Exception $e) {
+        $vars['message'] = $e->getMessage();
```

Comment:
We have a logger that can also used with different log levels, Its defined in `DefaultPlugin` so can be easily inherited. We can log the entire trace maybe? Helpful for development?

https://github.com/fossology/fossology/blob/da57da7adf118f05d8c7edeeb6a708949fab91a2/src/www/ui/page/AdminObligationFromCSV.php#L93